src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -142,10 +142,12 @@ void setupKernel(Path storeDirectory, String configFileName) throws InterruptedE
 
     @BeforeEach
     void beforeEach(ExtensionContext context) {
+        kernel = new Kernel();
         ignoreExceptionOfType(context, TLSAuthException.class);
         ignoreExceptionOfType(context, InterruptedException.class);
         ignoreExceptionOfType(context, DateTimeParseException.class);
         ignoreExceptionOfType(context, CrtRuntimeException.class);
+        ignoreExceptionOfType(context, SdkClientException.class);
```

Comment:
ignore this in _every_ test? Can we only ignore it when it will actually happen?